jsx/Form.js

```diff
@@ -380,7 +365,7 @@ SearchableDropdown.defaultProps = {
   options: {},
   strictSearch: true,
   label: '',
-  value: undefined,
+  value: null,
```

Comment:
not in **24.1**

_unclear if this will affect behaviour_

To be tested for adopting **24.1** — if problems occur, included as override and pushed to **main**